nntrainer/tensor/tensor.cpp

```diff
@@ -1076,7 +1076,7 @@ Tensor &Tensor::add(Tensor const &m, Tensor &output, float const alpha) const {
     auto f = [&](const BroadcastInfo &e, const float *buf, const float *m_buf,
                  float *out_buf) {
       if (e.strides[3] == 1 && strides[3] == 1 && strides[3] == 1 &&
-          alpha == 0) {
+          alpha == 1.f) {
```

Comment:
could you explain why this condition has changed?

Reply:
We have been using this code wrong.
`alpha` is an input scalar multiplier, so considering our total formula,

$$X = X  (op) Y * \alpha + Z * \beta$$

alpha == 0 is a nonsense. isn't it?

Reply:
As far as I am concerned, that is correct

Reply:
Nice catch!